spec/features/registered_customer_views_order_history_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.feature "Registered customer views orders", type: :feature do
+  before do
+    build_roles
+  end
+  let!(:user) { Fabricate(:user) }
+
+  scenario "successfully" do
+    order = Order.create(user_id: user.id)
+    user.orders << order
+
+    visit root_path
+    click_link "Sign in"
+    
+    fill_in "Email", with: user.email
+    fill_in "Password", with: user.password
+    click_button "Sign in"
+
+    expect(current_path).to eq(explore_path)
+
+    click_on "Orders"
+
+    expect(current_path).to eq(orders_path)
+    expect(page).to have_content("#{user.name}'s Orders")
+    expect(page).to have_content(order.id)
```

Comment:
is the order name or any other information about the order appearing on the page? maybe even add some expect statements to test that content to this test